Cargo.toml

```diff
@@ -12,28 +13,37 @@ categories = ["text-processing"]
 exclude = ["/.github/*", "/Cargo.lock.msrv"]
 
 [features]
-default = ["unicode", "perf"]
+default = ["unicode", "perf", "std"]
```

Comment:
Do note that this is unfortunately a breaking change.

Reply:
Thanks. I'm fine with this as we're still on 0.x.